eclipse-collections/src/main/java/org/eclipse/collections/impl/bag/immutable/ImmutableArrayBag.java

```diff
@@ -246,6 +246,25 @@ public T getLast()
     }
 
     @Override
+    public T getOnly()
+    {
+        Iterator<T> iterator = this.iterator();
+
```

Comment:
That is correct, and @motlin 's suggestion for this particular class is to avoid using iterator as similar to how `ImmutableArrayBag.getFirst()` does.